include/spirv-tools/libspirv.h

```diff
@@ -370,7 +370,7 @@ typedef enum {
 const char* spvTargetEnvDescription(spv_target_env env);
 
 // Creates a context object.  Returns null if env is invalid.
-spv_context spvContextCreate(spv_target_env env);
+spv_context spvContextCreate(spv_target_env env, bool permissive = false);
```

Comment:
if there's only 1 option, then there's no need to create an options word and use spvValidateBinaryWithOptions. We can just add spvValidateBinaryPermissive. 
(i.e. 
spvValidateBinary = the existing function would be the default
spvValidateBinaryPermissive = the permissive version of the validator)

I guess my question is: do we think there will be more command line options for the Validator executable in the future?